mne/simulation/tests/test_metrics.py

```diff
@@ -44,7 +44,8 @@ def test_metrics():
     stc_bad = stc2.copy().crop(0, 0.5)
     assert_raises(ValueError, source_estimate_quantification, stc1, stc_bad)
     stc_bad = stc2.copy()
-    stc_bad.times -= 0.1
+    stc_bad.tmin -= 0.1
+    stc_bad.update_times()
```

Comment:
but if you do `stc._data = stc._data[:, ::decim]` you hack a private attribute so there is nothing to expect in terms of safety. I think that asking users to use/know about update_times is way too much to ask.